src/LanguageServer/Protocol/Features/Diagnostics/DocumentAnalysisExecutor.cs

```diff
@@ -82,18 +82,6 @@ public async Task<IEnumerable<DiagnosticData>> ComputeDiagnosticsAsync(Diagnosti
         var document = textDocument as Document;
         RoslynDebug.Assert(document != null || kind == AnalysisKind.Syntax, "We only support syntactic analysis for non-source documents");
 
-        var loadDiagnostic = await textDocument.State.GetLoadDiagnosticAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
computation and returning of this diagnostic is pushed into FileContentLoadAnalyzer itself, instead of having to special case it here.